dev-client/src/translations/en.json

```
@@ -86,6 +86,17 @@
         "bedrock_label": "BEDROCK"
       }
     },
+    "notes": {
+      "add_note_label": "ADD NOTE",
+      "title": "Site Notes",
+      "add_title": "Site Note",
+      "note_by": "by",
+      "confirm_removal_title": "Delete note?",
+      "confirm_removal_body": "This action cannot be undone.",
```

If notes don't have names, find another way of identifying it, it. "remove the note for the site {{siteName}}"
```suggestion
      "confirm_removal_body": "Are you sure you want to remove the note {{name}}?",
```